src/PIL/ImageGrab.py

```diff
@@ -91,6 +91,8 @@ def grab(
                 args = ["gnome-screenshot", "-f"]
             elif shutil.which("spectacle"):
                 args = ["spectacle", "-n", "-b", "-f", "-o"]
+            elif shutil.which("grim"):
+                args = ["grim"]
```

Comment:
```suggestion
            elif shutil.which("grim"):
                args = ["grim"]
            elif shutil.which("spectacle"):
                args = ["spectacle", "-n", "-b", "-f", "-o"]
```
If there's no reason not to, let's make the choice behave alphabetically.